client/my-sites/earn/memberships/index.jsx

```diff
@@ -406,21 +406,23 @@ class MembershipsSection extends Component {
 			} );
 		} else if ( subscriber.plan.renew_interval === '1 year' ) {
 			return this.props.translate(
-				'Paying %(amount)s/year since %(formattedDate)s. Total of %(total)s.',
+				'Paying %(amount)s/year%(title)ssince %(formattedDate)s. Total of %(total)s.',
```

Comment:
Might be worth adding translator comments for these lines, especially where "title" could be ambiguous and the spacing changes depending on the variable.